contracts/ColonyTask.sol

```diff
@@ -232,10 +232,12 @@ contract ColonyTask is ColonyStorage, DSMath {
   taskExists(_id)
   taskNotFinalized(_id)
   {
+    uint8 initialRating = (_role == EVALUATOR) ? 50 : 0;
+
     tasks[_id].roles[_role] = Role({
       user: _user,
       rated: false,
-      rating: 0
+      rating: initialRating
```

Comment:
Yes, initially it was done just before calculating the reputation.

I don't know exactly how objections will work, but the reason i decided to do it this way is because maybe we would want to allow objections to the evaluator rating to be made before actual calculation of the reputation.

Reply:
You're right, objections will be done before reputation calculation but the evaluator rating won't be needed there still.